docs/jwst/emicorr/description.rst

```diff
@@ -0,0 +1,58 @@
+Description
+===========
+
+:Class: `jwst.emicorr.EmiCorrStep`
+:Alias: emicorr
+
+Overview
+--------
+The ``emicorr`` step corrects for known noise patterns in the raw MIRI data.
+The majority of the MIRI subarrays have an 390 Hz or other electromagnetic
+interference (EMI) noise pattern in the raw data. The known frequencies to
+correct for are in the EMI reference file, under the key ``frequencies``.
+The effect of these EMI frequencies is to imprint each into the rate
+images. For short integrations in ``LRSSLITLESS`` the correlated noise from
+this is quite apparent in the rate images. For longer integrations the net
+effect is to increase the read noise by about 20\%.
+
+The process to do the correction is the following (repeated
```

Comment:
The docs build in the CI tests indicates it's failing at line 18 of this file, but I don't see anything obviously wrong with it. Could it be the use of the escaped percent sign at the end of the previous line that might be tripping it up?

Reply:
not sure. Added a white space before escaping, hopefully that does it